src/context.rs

```diff
@@ -690,6 +691,40 @@ mod tests {
         assert_eq!(t.get_fresh_msgs().await.unwrap().len(), 9); // claire is counted again
     }
 
+    #[async_std::test]
+    async fn test_get_fresh_msgs_and_muted_until() {
```

Comment:
Corner case of muted forever is not tested, otherwise looks good.

Reply:
`MuteDuration::Forever` is tested implicitly by `test_get_fresh_msgs_and_muted_chats()`, however, it makes sense to also test it explicitly here. i'll change that.